Build failure on OpenBSD-current.

```
   Compiling nix v0.15.0 (/home/edd/source/nix)
error[E0308]: mismatched types
   --> src/sys/statfs.rs:111:9
    |
110 |     pub fn optimal_transfer_size(&self) -> i32 {
    |                                            --- expected `i32` because of return type
111 |         self.0.f_iosize
    |         ^^^^^^^^^^^^^^^ expected i32, found u32
help: you can convert an `u32` to `i32` and panic if the converted value wouldn't fit
    |
111 |         self.0.f_iosize.try_into().unwrap()
    |

error[E0308]: mismatched types
   --> src/sys/statfs.rs:392:9
    |
391 |     pub fn files_free(&self) -> i64 {
    |                                 --- expected `i64` because of return type
392 |         self.0.f_ffree
    |         ^^^^^^^^^^^^^^ expected i64, found u64
help: you can convert an `u64` to `i64` and panic if the converted value wouldn't fit
    |
392 |         self.0.f_ffree.try_into().unwrap()
    |

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0308`.
error: Could not compile `nix`.
```

This is using the master branch, but the failure also exists on 0.15.0, but not on 0.14.1.

Thanks

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased] - ReleaseDate
+### Added
+### Changed
+### Fixed
+
+- Fixed the build for OpenBSD
+  (#[1168](https://github.com/nix-rust/nix/pull/1168))
+
+### Removed
 
 ## [0.16.0] - 1 December 2019
 ### Added

--- a/src/sys/statfs.rs
+++ b/src/sys/statfs.rs
@@ -106,8 +106,14 @@ impl Statfs {
     }
 
     /// Optimal transfer block size
-    #[cfg(any(target_os = "ios", target_os = "macos", target_os = "openbsd"))]
+    #[cfg(any(target_os = "ios", target_os = "macos"))]
     pub fn optimal_transfer_size(&self) -> i32 {
+        self.0.f_iosize
+    }
+
+    /// Optimal transfer block size
+    #[cfg(target_os = "openbsd")]
+    pub fn optimal_transfer_size(&self) -> u32 {
         self.0.f_iosize
     }
 
@@ -118,7 +124,10 @@ impl Statfs {
     }
 
     /// Optimal transfer block size
-    #[cfg(all(target_os = "linux", target_env = "musl"))]
+    #[cfg(any(
+        target_os = "android",
+        all(target_os = "linux", target_env = "musl")
+    ))]
     pub fn optimal_transfer_size(&self) -> libc::c_ulong {
         self.0.f_bsize
     }
@@ -126,12 +135,6 @@ impl Statfs {
     /// Optimal transfer block size
     #[cfg(all(target_os = "linux", not(any(target_arch = "s390x", target_env = "musl"))))]
     pub fn optimal_transfer_size(&self) -> libc::c_long {
-        self.0.f_bsize
-    }
-
-    /// Optimal transfer block size
-    #[cfg(target_os = "android")]
-    pub fn optimal_transfer_size(&self) -> libc::c_ulong {
         self.0.f_bsize
     }
 
@@ -375,7 +378,13 @@ impl Statfs {
     }
 
     /// Free file nodes in filesystem
-    #[cfg(any(target_os = "ios", target_os = "macos", target_os = "android"))]
+    #[cfg(any(
+            target_os = "android",
+            target_os = "ios",
+            all(target_os = "linux", target_env = "musl"),
+            target_os = "macos",
+            target_os = "openbsd"
+    ))]
     pub fn files_free(&self) -> u64 {
         self.0.f_ffree
     }
@@ -387,14 +396,8 @@ impl Statfs {
     }
 
     /// Free file nodes in filesystem
-    #[cfg(any(target_os = "freebsd", target_os = "openbsd"))]
+    #[cfg(target_os = "freebsd")]
     pub fn files_free(&self) -> i64 {
-        self.0.f_ffree
-    }
-
-    /// Free file nodes in filesystem
-    #[cfg(all(target_os = "linux", target_env = "musl"))]
-    pub fn files_free(&self) -> u64 {
         self.0.f_ffree
     }
 

